### ISSUE TITLE
accessibility: cannot leave the edit box with a screen reader without going to top or bottom of the page or similar

### ISSUE THREAD

**Issue by lilmike** [thelounge/thelounge#935]
Hi,
I am using orca on linux, and when in the edit box for input to the channel, you can't arrow or tab around with the screen reader without going to the top or bottom of the page (ctrl+end/home, etc) first. This causes you to have to scroll through the whole page to get what you want in most cases if you have just typed a message. If this could be fixed that would be great!
Thanks,
-Michael.

**Comment by lilmike** [comment]
Hi,
First of all, there's one thing that needs to be understood before you can understand how this works:
Most screen readers (including orca) have something they call either forms mode, focus mode, or similar. This basically allows you to type into an edit field, move in a combo box, etc, without having the keys do anything else. For example, assuming the feature is turned on, h will cycle through headings on the webpage. This is obviously not desirable if you're trying to type into an edit box. Now for what happens:

When you enter focus mode on the chat edit box, you can type, etc. However, when You leave it, arrows and shift-tab (and tab) still go to the edit box. I will have to see if there's something y'all can do about it or if this is an orca issue, but that's basically what happens. Even though focus mode is off, and, presumably, the edit field is not in focus, the keys are stil intercepted.
I'll let you know what I find out.
-Michael.

**Comment by xPaw** [comment]
I believe this should be much better in 3.0-pre5. @lilmike if you still use The Lounge, can you try it out?

**Comment by xPaw** [comment]
Going to close as I believe this is now fixed and due to no feedback.
